server/constants/vars.js

```diff
@@ -62,3 +64,49 @@ export const ALL_SUBJECTS_MONGO_PROJECTION = {
   _id: 0,
   subject: 1,
 }
+
+export const DEFAULT_CONFIG_AT_SIGNUP = (uid) => ({
+  owner: uid,
+  config: defaultUserConfig,
+  name: 'default',
+  type: 'matrix',
+  readers: [uid],
+  created: new Date().toUTCString(),
+})
+
+export const SIGN_IN_MONGO_PROJECTION = {
+  _id: 0,
+  uid: 1,
+  display_name: 1,
+  acl: 1,
+  role: 1,
+  icon: 1,
+  mail: 1,
+  access: 1,
+  account_expires: 1,
+}
+
+export const INITIAL_ROLE = 'member'
+
+export const SIGN_UP_UPDATE_DATA = (overrides) => ({
+  display_name: '',
+  title: '',
+  department: '',
+  company: '',
+  mail: '',
+  member_of: '',
+  bad_pwd_count: 0,
+  lockout_time: 0,
+  last_logoff: Date.now(),
```

Comment:
should the logoff time be populated by default?

Reply:
When a user first signs up they don't get access to dpdash they get kicked to the home screen and an admin gives them access later.